video/graphics.h

```diff
@@ -245,22 +245,27 @@ Point * getGraphicsCursor() {
 //
 void setGraphicsOptions(uint8_t mode) {
 	auto colourMode = mode & 0x03;
+	fabgl::PaintOptions p = gpo;
+
 	canvas->setClippingRect(graphicsViewport);
 	switch (colourMode) {
 		case 0: break;	// move command
 		case 1: {
 			// use fg colour
 			canvas->setPenColor(gfg);
-			canvas->setBrushColor(gfg);
```

Comment:
brush colour is for fill operations, which are set with a separate function call